pom.xml

```diff
@@ -282,6 +271,17 @@ limitations under the License.]]>
           </instructions>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.sonatype.central</groupId>
+        <artifactId>central-publishing-maven-plugin</artifactId>
+        <version>0.7.0</version>
```

Comment:
Probably better to also change the plugin [above](https://github.com/datastax/native-protocol/pull/50/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R100-R104) in the `pluginManagement` section, declare the version there (and remove the nexus one).

Reply:
Excellent catch @emerkle826 ... I think you're exactly right!